test/update-db.test.js

```diff
@@ -133,3 +133,56 @@ if (!NODE_8 && (isInstalled('pnpm') || process.env.CI)) {
     expect(lock).toContain(`/caniuse-lite/${ caniuse.version }:`)
   })
 }
+
+it('shows target browser changes', async () => {
+  let dir = await chdir('browserslist-diff',
+    'package.json', 'package-lock.json')
+
+  expect(runUpdate()).toEqual(
+    'Current version: 1.0.30001030\n' +
+    `New version: ${ caniuse.version }\n` +
+    'Removing old caniuse-lite from lock file\n' +
+    'Installing new caniuse-lite version\n' +
+    '$ npm install caniuse-lite\n' +
+    'Cleaning package.json dependencies from caniuse-lite\n' +
+    '$ npm uninstall caniuse-lite\n' +
+    'caniuse-lite has been successfully updated\n' +
+    '\n' +
+    'Target browser changes:\n' +
+    '- chrome 83\n' +
+    '+ chrome 86\n' +
+    '- edge 84\n' +
+    '+ edge 86\n' +
+    '- firefox 80\n' +
```

Comment:
Maybe it's better to show difference for the same browser like:
```
- firefox 80, 79, 78
+ firefox 82, 81, 68
```
or 
```
- firefox 80 79 78
+ firefox 82 81 68
```

Reply:
I haven't implemented version sort, cause it can be complicated, e.g. to compare safari version like 12.0-12.1.
But if it's needed a simple one, e.g. based on the major version can be introduced

Reply:
Let’s keep as it and improve later